static/js/util/google_analytics.js

```diff
@@ -37,3 +37,11 @@ export const initGA = () => {
     ga.initialize(SETTINGS.gaTrackingID, { debug: debug })
   }
 }
+
+export const setCustomDimension = (dimension: string, value: string) => {
+  if (dimension) {
+    const dimensionObject = {}
```

Comment:
In ES6 you can specify keys like this: `const dimensionObject = { [dimension]: value }`

Reply:
thanks!